contracts/tokens/SecurityToken.sol

```diff
@@ -289,6 +292,15 @@ contract SecurityToken is ERC20, ERC20Detailed, Ownable, ReentrancyGuard, Securi
         dataStore = _dataStore;
     }
 
+    /**
+    * @notice Allows owner to change token name
+    * @param _name new name of the token
+    */
+    function changeName(string calldata _name) external onlyOwner {
```

Comment:
Should we add the sanity check for the `_name`? I know it only called by the owner but still, the owner can do the mistake

Reply:
Deploying nameless ST is allowed maybe we should allow changing name to nameless as well?

What if faceless men want to deploy their own ST? (useless GOT reference)

Reply:
I am not sure about this. Because they need to submit the investment report to SEC so there is no chance to work as nameless. Even if they have private equity round issuer needs to come in front of its investors. 
BTW I like the reference. Winter is here !!

Reply:
I don't see much value in having a contract level check. Even if we insisted that _name.length > 0, they could still just put something silly in (i.e. " "). The dApp can enforce some sensible scheme if needed, otherwise if the faceless men want to go wild they can do.